src/main/java/com/ratifire/devrate/controller/HandlerException.java

```diff
@@ -101,6 +109,54 @@ public void handleNoFoundExceptions() {
   public void handleAuthenticationExceptions() {
   }
 
+  /**
+   * Handles the logout exceptions by returning an HTTP status 401.
+   */
+  @ResponseStatus(HttpStatus.UNAUTHORIZED)
+  @ExceptionHandler({LogoutException.class})
+  public void handleLogoutExceptions() {
+  }
+
+  /**
+   * Handles the TokenExpired exceptions by returning an HTTP status 419.
+   */
+  @ExceptionHandler(TokenExpiredException.class)
+  public void handleTokenExpiredException(HttpServletResponse response) {
+    response.setStatus(EXPIRED_TOKEN_HTTP_STATUS);
+  }
+
+  /**
+   * Handles the PasswordReset exceptions by returning an HTTP status 400.
+   */
+  @ResponseStatus(HttpStatus.BAD_REQUEST)
+  @ExceptionHandler({PasswordResetException.class})
+  public void handlePasswordResetExceptionExceptions() {
+  }
+
+  /**
+   * Handles the RefreshToken exceptions by returning an HTTP status 400.
+   */
+  @ResponseStatus(HttpStatus.BAD_REQUEST)
+  @ExceptionHandler({RefreshTokenException.class})
+  public void handleRefreshTokenExceptions() {
+  }
+
+  /**
+   * Handles the UserRegistration exceptions by returning an HTTP status 400.
+   */
+  @ResponseStatus(HttpStatus.BAD_REQUEST)
+  @ExceptionHandler({UserRegistrationException.class})
+  public void handleUserRegistrationExceptions() {
+  }
```

Comment:
I'm not sure is it a good idea or not, but we can decrease lines of code by using smth like this:
```
@ResponseStatus(HttpStatus.BAD_REQUEST)
  @ExceptionHandler({UserRegistrationException.class, 
  RefreshTokenException.class, 
  PasswordResetException.class })
  public void handleAuthenticationExceptions() {
  }
```